pyaedt/modeler/cad/Primitives.py

```diff
@@ -275,45 +587,4908 @@ def version(self):
+        name : str
+            Name of the coordinate system. The default is ``None``.
+        mode : str, optional
+            Definition mode. Options are ``"view"``, ``"axis"``,
+            ``"zxz"``, ``"zyz"``, and ``"axisrotation"``. The default
+            is ``"axis"``.  Enumerator ``pyaedt.generic.constants.CSMODE`` can be used.
+
+            * If ``mode="view"``, specify ``view``.
+            * If ``mode="axis"``, specify ``x_pointing`` and ``y_pointing``.
+            * If ``mode="zxz"`` or ``mode="zyz"``, specify ``phi``, ``theta``, and ``psi``.
+            * If ``mode="axisrotation"``, specify ``theta`` and ``u``.
+
+            Parameters not needed by the specified mode are ignored.
+            The default mode, ``"axis"``, is a coordinate system parallel to the
+            global coordinate system centered in the global origin.
+
+        view : str, int optional
+            View for the coordinate system if ``mode="view"``. Options
+            are ``"XY"``, ``"XZ"``, ``"XY"``, ``"iso"``, ``None``, and
+            ``"rotate"`` (obsolete). The default is ``"iso"``.
+            Enumerator ``pyaedt.generic.constants.VIEW`` can be used.
+
+            .. note::
+              For backward compatibility, ``mode="view"`` and ``view="rotate"`` are the same as
+              ``mode="axis"``. Because the "rotate" option in the "view" mode is obsolete, use
+              ``mode="axis"`` instead.
+
+        x_pointing : list, optional
+            List of the ``[x, y, z]`` coordinates specifying the X axis
+            pointing in the global coordinate system if ``mode="axis"``.
+            The default is ``[1, 0, 0]``.
+        y_pointing : list, optional
+            List of the ``[x, y, z]`` coordinates specifying the Y axis
+            pointing in the global coordinate system if ``mode="axis"``.
+            The default is ``[0, 1, 0]``.
+        phi : float, optional
+            Euler angle phi in degrees if ``mode="zxz"`` or ``mode="zyz"``.
+            The default is ``0``.
+        theta : float, optional
+            Euler angle theta or rotation angle in degrees if ``mode="zxz"``,
+            ``mode="zyz"``, or ``mode="axisrotation"``. The default is ``0``.
+        psi : float, optional
+            Euler angle psi in degrees if ``mode="zxz"`` or ``mode="zyz"``.
+            The default is ``0``.
+        u : list
+            List of the ``[ux, uy, uz]`` coordinates for the rotation axis
+            if ``mode="zxz"``. The default is ``[1, 0, 0]``.
+
+        Returns
+        -------
+        :class:`pyaedt.modeler.Modeler.CoordinateSystem`
+            Created coordinate system.
+
+        References
+        ----------
+
+        >>> oEditor.CreateRelativeCS
+        """
+        if name:
+            cs_names = [i.name for i in self.coordinate_systems]
+            if name in cs_names:
+                raise AttributeError("A coordinate system with the specified name already exists!")
```

Comment:
```suggestion
                raise AttributeError("A coordinate system with the specified name already exists.")
```
Avoid using exclamation points in all techical documentation.